templates/common/_base/units/nodeip-configuration.service.yaml

```diff
@@ -1,7 +1,7 @@
 # NOTE: When making changes to this service, be sure they are replicated in the
 # VSphere-specific service in templates/common/[vsphere|kubevirt]/units
```

Comment:
Do we support SNO on VSphere and Kubevirt? If so we should probably add this logic to those units as well.

Reply:
That's a good point, we do to my knowledge. I had left the [kubevirt](https://github.com/openshift/machine-config-operator/blob/master/templates/common/kubevirt/units/nodeip-configuration.service.yaml#L2) config alone since it's enabled by default. The VSpheare [the comment on the code](https://github.com/openshift/machine-config-operator/blob/master/templates/common/vsphere/units/nodeip-configuration-vsphere-upi.service.yaml#L7-L9) made it sound like there's something else picking an IP for UPI deployments? If that's not the case then I can add the logic in there specific for SingleNode deployments as well.

Reply:
I don't have a way to stand up a SNO cluster on VSphere UPI to validate this 100%, but that does seem to be the case, pulled this info from the currently running VSphere UPI HA which seems correct. 

```
[root@control-plane-0 /]# cat /etc/systemd/system/kubelet.service.d/20-nodenet.conf 
[Service]
Environment="KUBELET_NODE_IP=<correct_redacted_ip>" "KUBELET_NODE_IPS=<correct_redacted_ip>"
```

Reply:
Oh actually, apologies Ben, the wording in the comment and the render helper function [onPremPlatformAPIServerInternalIPs](https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/template/render.go#L589-L594) confused me, it seems like for UPI this will be enabled. I'll add the same `or` logic to enable it for external SNO like the other  to cover our bases.

Reply:
Updated, thanks for the advice!